.travis.yml

```diff
@@ -3,21 +3,17 @@ sudo: false
 language: python
 
 python:
-  - 3.7
-  - 3.6
   - 3.5
+  - 3.8
 
 env:
-  - TOX_ENV=django3
+  - TOX_ENV=django30
   - TOX_ENV=django22
-  - TOX_ENV=django21
-  - TOX_ENV=django20
-  - TOX_ENV=django111
 
 matrix:
   exclude:
     - python: 3.5
-      env: TOX_ENV=django3
+      env: TOX_ENV=django30
```

Comment:
as per Jeremy, we shouldn't add Django 3.0 in Travis builds just yet

Reply:
It was already added, I just changed it from 3 to 30. Should I remove it?

Reply:
as described [here](https://openedx.atlassian.net/wiki/spaces/AC/pages/1368064245/Post-Upgrade+Test+Matrix+Cleanup), it shouldn't have been added in the first place